pom.xml

```diff
@@ -24,6 +24,12 @@
     <module>RosettaJdbi3</module>
   </modules>
 
+  <properties>
+    <dep.jackson.version>2.9.9</dep.jackson.version>
+    <dep.jackson-databind.version>2.9.9.3</dep.jackson-databind.version>
+
```

Comment:
nit: extra newline. Do we still want to leave in the jackson version upgrade?

Reply:
I don't think so, i was just gonna remove it once we were good with this PR.